cirq/ops/common_gates_test.py

```diff
@@ -540,16 +540,26 @@ def test_measurement_channel():
     np.testing.assert_allclose(
             cirq.channel(cirq.MeasurementGate(1)),
             (np.array([[1, 0], [0, 0]]), np.array([[0, 0], [0, 1]])))
+    # yapf: disable
```

Comment:
Why is yapf wrong?

Reply:
because formatting a big 4 by 4 array in a readable format is way more readable than cludging it together.